overview.dd

```diff
@@ -1044,23 +1067,25 @@ $(SECTION2 Sample D Program (sieve.d),
 
 import std.stdio;
 
-bool[8191] flags;
-
-int main()
+void main()
 {
-    int i, count, prime, k, iter;
+    size_t i, count, prime, k;
+    bool[8191] flags;
 
-    writeln("10 iterations");
-    for (iter = 1; iter <= 10; iter++)
+    "10 iterations".writeln;
```

Comment:
Please no, we don't use UFCS for writeln.

`writefln` and `format` are a bit different in that the first argument is special, but I don't think there is much point in confusing newcomers here.